src/shared/components/EventMarkers.test.tsx

```diff
@@ -75,15 +75,15 @@ describe('EventMarkers', () => {
     )
 
     expect(queryByTestId(`eventMarker`)).toBeNull()
-    expect(getByTestId(`event-marker-vis-icon-ok`).className).toContain(
-      'eye-closed'
+    expect(getByTestId(`event-marker-vis-icon-ok`).className).toMatch(
```

Comment:
Why is this one changing? In `ShareOverlay`, `EyeNew` got replaced with `EyeOpen`, but I don't see anywhere where `Eye-Closed` got swapped out. Is it in the SVG file - or anywhere else?

Reply:
Nevermind - I see this in the prior clockface PR adding `EyeClosed`.

Reply:
It may just be an icon added to the clockface set to mirror the open. In case we ever use the eyeopen as a toggle